docs/training/apis-howto/api-pytorch-ug.rst

```diff
@@ -505,6 +505,40 @@ you find that the built-in context.DataLoader() does not support your use case.
 
 See the :mod:`determined.pytorch.samplers` for details.
 
+Profiling
+---------
+
+Determined provides integrated support for the native PyTorch profiler, `torch-tb-profiler
```

Comment:
Determined provides support for the native PyTorch profiler....

(is "integrated" assumed since having integrated it we are showing them how to use it?)